.github/workflows/tests.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       #for each of the following versions of PHP, with and without --prefer-lowest
       matrix:
-        php-versions: ['5.5', '5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0', '8.1', '8.2']
+        php-versions: ['7.2.5', '7.3', '7.4', '8.0', '8.1', '8.2']
```

Comment:
One should test on 7.2.5 and also 7.2 (7.2 is not 7.2.0, but the latest 7.2.x version)